src/chromo/models/dpmjetIII.py

```diff
@@ -213,10 +213,11 @@ def _generate(self):
 
 class DpmjetIII191(DpmjetIIIRun):
     _version = "19.1"
+    _projectiles = standard_projectiles | Nuclei() | {3322, 3312, 3222, 3122, 3112, 311}
     _library_name = "_dpmjetIII191"
 
 
-class DpmjetIII193(DpmjetIIIRun):
+class DpmjetIII193(DpmjetIII191):
```

Comment:
If you put the extra code into DpmjetIIIRun, you can revert this change.

Reply:
ok